Readme.md

```diff
@@ -34,6 +34,7 @@ With the following commands, you can install the latest version of each product:
 # Formulae
 brew install opslevel/tap/cli
 brew install opslevel/tap/kubectl
+brew install opslevel/tap/opslevel-runner
```

Comment:
don't advertise this yet - the runner is not for consumption by the public just yet.  Only OpsLevel can setup instances of the runner.  Long term yes we want customers to beable to do this but since they cannot yet there is no reason to advertise this yet.